inst/templates/Makevars

```diff
@@ -8,7 +8,9 @@ all: C_clean
 $(SHLIB): $(STATLIB)
 
 $(STATLIB):
-	cargo build --lib --release --manifest-path=./rust/Cargo.toml --target-dir $(TARGET_DIR)
+	# In some environment, ~/.cargo/bin is not included in PATH, so we need to set it here to make sure
```

Comment:
'In some environment*s*'. Also, it seems the end of the phrase is missing. I suggest 'so we need to set it here to ensure cargo can be invoked'

Reply:
Thanks, I'll use the phrase.